utils/cligen/Makefile

```
@@ -0,0 +1,66 @@
+#
+# Copyright (C) 2020-2023 Olof Hagsand and Rubicon Communications, LLC(Netgate)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cligen
+PKG_VERSION:=6.4.0
+PKG_RELEASE:=1
+PKG_MAINTAINER:=Olof Hagsand <olof@hagsand.se>, Philip Prindeville <philipp@redfish-solutions.com>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE.md
+
+PKG_SOURCE_URL:=https://github.com/clicon/$(PKG_NAME)/archive/refs/tags
```

```suggestion
PKG_SOURCE_URL:=https://github.com/clicon/cligen/archive/refs/tags
```

Not sure, where you seen that this variable PKG_NAME should be in the PKG_SOURCE_URL. It looks like to me that it is some bad habbit and in some examples should be corrected.

---

I just ran:

```
grep -r PKG_SOURCE_URL | grep PKG_NAME
```

and got 86 hits.  It's apparently not that rare.  What's the problem with it?  I like it because it's idiomatic and avoids duplication.